From 05c63817ac715fdcf9065568d796b93d97c0be7f Mon Sep 17 00:00:00 2001
From: Jiffin Tony Thottan <jthottan@redhat.com>
Date: Tue, 2 May 2017 14:06:00 +0530
Subject: [PATCH 62/74] glusterd/ganesha : add proper NULL check in
 manage_export_config

Change-Id: I872b2b6b027f04e61f60ad85588f50e1ef2f988c
Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>
Reviewed-on: https://review.gluster.org/17150
Smoke: Gluster Build System <jenkins@build.gluster.org>
Reviewed-by: soumya k <skoduri@redhat.com>
NetBSD-regression: NetBSD Build System <jenkins@build.gluster.org>
Reviewed-by: Kaleb KEITHLEY <kkeithle@redhat.com>
CentOS-regression: Gluster Build System <jenkins@build.gluster.org>
---
 xlators/mgmt/glusterd/src/glusterd-ganesha.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-ganesha.c b/xlators/mgmt/glusterd/src/glusterd-ganesha.c
index cd591df..7ba25ee 100644
--- a/xlators/mgmt/glusterd/src/glusterd-ganesha.c
+++ b/xlators/mgmt/glusterd/src/glusterd-ganesha.c
@@ -454,7 +454,7 @@ manage_export_config (char *volname, char *value, char **op_errstr)
                          CONFDIR, value, volname, NULL);
         ret = runner_run(&runner);
 
-        if (ret && !(*op_errstr))
+        if (ret && op_errstr)
                 gf_asprintf (op_errstr, "Failed to create"
                             " NFS-Ganesha export config file.");
 
-- 
1.8.3.1

